src/main/solr/dssolr/conf/schema.xml

```diff
@@ -582,6 +582,11 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     www.example.com/streaming/bart-access-copies-tv/00/00/00/000000e1-0000-462a-a2b4-7488244fcca7/playlist.m3u8"?>
   </field>
 
+  <field name="date_modified" type="pdate" stored="true">
+    <?description Milliseconds since 1st of January 1970 (epoch) with three added digits representing the last time a record has been modified?>
```

Comment:
That mighe be true beneath the hood, but conceptually it is now simply a point in time (as for `startTime` and `endTime`. And the three digits are gone :-)

Also, while the example is technically correct as a rendering of the `pdate`, so is "November 29., 2023, 6:45PM". It is just a representation.

Solr has no explicit concept of timezones, i.e. all time is Zulu time. That is why the default rendering, the one returned from the common Solr request, is `2023-11-29 14:45:49Z`. The example should follow that pattern.